src/components/ui/FormErrorMessage.tsx

```diff
@@ -0,0 +1,11 @@
+const FormErrorMessage = ({ error }: { error?: string }) => {
+  return error ? (
+    <p role='alert' className='text-sm text-red-600'>
```

Comment:
based on the surrounding font sizes, I think looks a bit nicer

```suggestion
    <p role='alert' className='text-xs mt-1 text-red-600'>
```

Reply:
https://github.com/ScopeLift/cove-frontend/pull/9/commits/572ea46438850bf1894b2543e5a0cc8aed9da7b0 Update CSS